src/pages/become-a-sponsor.js

```diff
@@ -118,6 +118,29 @@ export default () => {
             </tr>
           </thead>
           <tbody>
+          <tr>
+              <td>Limited to</td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  3
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️5
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  &#8734;
+                </span>
+              </td>
+            </tr>
```

Comment:
I think I would rather have them reach out or we cross of the row when sold out than to display the counts.

Reply:
I'm going to remove this row then. And we can add a call to action elsewhere.